src/lib/auth/databaseKeys.mjs

```diff
@@ -50,7 +50,7 @@ export class DatabaseKeys {
     // argv.secret comes from flag, config, or FAUNA_SECRET
     if (argv.secret) {
       key = argv.secret;
-      keySource = "--secret";
+      keySource = "user";
```

Comment:
So this is this describing that the source of the key is a user-provided value? Not that specifically the `--user` arg was provided, correct?

Reply:
correct